CONTRIBUTING.md

```diff
@@ -180,14 +180,13 @@ npm run build
 npm start
 ```
 
-If there are errors building the packages, run with `DEBUG=cypress:*`
-option to see more details.
+If there are errors building the packages, prefix the commands with `DEBUG=cypress:*` to see more details.
 
-This outputs a lot of debugging lines. To focus on an individual module run with `DEBUG=cypress:launcher` for instance.
+This outputs a lot of debugging lines. To focus on an individual module, run with `DEBUG=cypress:launcher` for instance.
 
-When running `npm start` this routes through the CLI and eventually calls `npm run dev` with the proper arguments. This enables Cypress day to day development to match the logic of the built binary + CLI integration.
+When running `npm start` this routes through the CLI and eventually calls `npm run dev` with the proper arguments. This enables Cypress day-to-day development to match the logic of the built binary + CLI integration.
```

Comment:
"day-to-day" when used as an adjective (day-to-day work)
"day to day" as an adverb (I work out day to day)

Reply:
never knew about this one 💯